### PR TITLE
Cleaning up tuning mxnet for performance

### PR DESCRIPTION
**Pull request by sandeep-krishnamurthy**
* Cleaned up issues with performance tuning How Tos.
* Verified compiling with profiler flag, setting environment variables for profiling and visualizing profile output in the browser.

@piiswrong @nswamy @madjam - Can you please review and merge the change?

### CODE REVIEW

File: docs/how_to/perf.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Some Tips for Improving MXNet Performance
-Even fixing the training or deployment environment and parallelization scheme,
-a number of configuration settings and data-handling choices can impact performance.
-In this document, we address some tips for improving _MXNet_ performance.
+Even after fixing the training or deployment environment and parallelization scheme,
+a number of configuration settings and data-handling choices can impact the _MXNet_ performance.
```
Comment (piiswrong) [inline docs/how_to/perf.md]: why the?

**Comment by sandeep-krishnamurthy** [comment]
Fixed review comments by @piiswrong

File: docs/how_to/perf.md [hunk 2]
```diff
@@ -243,7 +244,7 @@ The `mode` parameter can be set to
 * `symbolic` to only include symbolic operations
 * `all` to include all operations
 
-After the program finishes, navigate to chrome://tracing in a Chrome browser and load the `.json` file output by the profiler to inspect the results.
+After the program finishes, navigate to your browser's tracing (Example - chrome://tracing in a Chrome browser) and load the `profile_output.json` file, output by the profiler, to inspect the results.
```
Comment (madjam) [inline docs/how_to/perf.md]: I thought the previous description was pretty good - the two commas you added are probably not needed.

**Comment by sandeep-krishnamurthy** [comment]
Fixed review comments by @madjam

**Pull request merged**